core/templates/pages/exploration-player-page/exploration-player-page.component.html

```diff
@@ -1,9 +1,29 @@
 <div>
-  <background-banner></background-banner>
+  <div *ngIf="isLoadingExploration" class="loader-container">
+    <p>Loading...</p>
+    <mat-spinner></mat-spinner>
+  </div>
```

Comment:
Can you please use the same loading mesage that is shown just before this loader? Having two different types of loaders is visually inconsistent for the users.

Reply:
Yes @kapilag96, I have changed it with the same loading message.
I updated the video of the same in the above comment. PTAL